wallet/package.json

```diff
@@ -4,58 +4,117 @@
     "comlink-loader": "^2.0.0",
     "crypto-js": "^4.1.1",
+    "css-loader": "4.3.0",
+    "dotenv": "8.2.0",
+    "dotenv-expand": "5.1.0",
+    "eslint": "^7.11.0",
+    "eslint-config-react-app": "^6.0.0",
+    "eslint-plugin-flowtype": "^5.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-testing-library": "^3.9.2",
+    "eslint-webpack-plugin": "^2.5.2",
     "ethereumjs-wallet": "^1.0.2",
+    "file-loader": "6.1.1",
     "font-awesome": "4.7.0",
+    "fs-extra": "^9.0.1",
     "general-number": "^1.0.1",
+    "html-webpack-plugin": "4.5.0",
     "idb": "^7.0.0",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "26.6.0",
+    "jest-circus": "26.6.0",
+    "jest-resolve": "26.6.0",
+    "jest-watch-typeahead": "0.6.1",
     "js-sha3": "^0.8.0",
     "keccak": "^3.0.2",
     "lottie-react": "^2.2.1",
+    "mini-css-extract-plugin": "0.11.3",
+    "optimize-css-assets-webpack-plugin": "5.0.4",
+    "pnp-webpack-plugin": "1.6.4",
+    "postcss-flexbugs-fixes": "4.2.1",
+    "postcss-loader": "3.0.0",
+    "postcss-normalize": "8.0.1",
+    "postcss-preset-env": "6.7.0",
+    "postcss-safe-parser": "5.0.2",
+    "prompts": "2.4.0",
     "prop-types": "^15.7.2",
-    "qrcode.react": "^3.0.0",
+    "qrcode.react": "^2.0.0",
     "queue": "^6.0.2",
     "react": "^17.0.2",
+    "react-app-polyfill": "^2.0.0",
     "react-bootstrap": "^2.1.1",
     "react-copy-to-clipboard": "^5.0.4",
+    "react-dev-utils": "^11.0.3",
     "react-dom": "^17.0.2",
     "react-icons": "^4.3.1",
     "react-redux": "7.2.5",
+    "react-refresh": "^0.8.3",
     "react-responsive": "^9.0.0-beta.6",
     "react-router-dom": "5.3.0",
     "react-to-webcomponent": "^1.5.1",
     "react-toastify": "^8.2.0",
     "redux": "4.1.1",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
+    "resolve": "1.18.1",
+    "resolve-url-loader": "^3.1.2",
     "safe-buffer": "^5.2.1",
     "sass": "^1.49.7",
-    "typescript": "4.4.4",
+    "sass-loader": "^10.0.5",
+    "semver": "7.3.2",
+    "style-loader": "1.3.0",
+    "styled-components": "^5.3.5",
+    "systemjs-webpack-interop": "^2.3.7",
+    "terser-webpack-plugin": "4.2.3",
+    "ts-pnp": "1.2.0",
+    "typescript": "^4.5.5",
+    "url-loader": "4.1.1",
     "wasm-loader": "^1.3.0",
     "web-vitals": "^2.1.0",
     "web3": "^1.6.0",
-    "webpack-manifest-plugin": "^4.1.1",
+    "webpack": "4.44.2",
+    "webpack-dev-server": "3.11.1",
+    "webpack-manifest-plugin": "2.2.0",
+    "workbox-webpack-plugin": "5.1.4",
     "zokrates-js": "^1.0.39"
   },
   "scripts": {
-    "start": "LOCAL_PROPOSER=false USE_STUBS=true craco start",
+    "start": "LOCAL_PROPOSER=true PUBLIC_URL='/nightfall/' USE_STUBS=true node scripts/start.js",
     "start:docker": "REACT_APP_ENVIRONMENT=Docker PORT=3010 craco start",
```

Comment:
craco commands should be replaced with the scripts

Reply:
![image](https://user-images.githubusercontent.com/73957838/164483885-7c75128e-a01b-49c5-ace2-f7edf5255ae6.png)
**In this branch just have with the script**